tests/suites/test_suite_cipher.gcm.data

```diff
@@ -1,3 +1,15 @@
+AES 128 GCM Decrypt empty buffer
```

Comment:
Contrary to what I wrote earlier, GCM is already covered by `enc_dec_buf` with a size of 0. In our AEAD API, passes the ciphertext-sans-tag as ciphertext and passes the tag separately, so `enc_dec_buf` with a 0-size plaintext exercises 0-size ciphertext well.

Reply:
Why did you remove AES-GCM but keep Camellia-GCM and ARIA-GCM?